subsys/net/lib/azure_iot_hub/src/azure_iot_hub_dps.c

```diff
@@ -968,26 +968,19 @@ int azure_iot_hub_dps_hostname_delete(void)
 	int err;
 
 	err = settings_delete(DPS_SETTINGS_KEY "/" DPS_SETTINGS_HOSTNAME_KEY);
-	if (err == -ENOENT) {
-		LOG_DBG("No hostname was stored");
-		goto exit_on_success;
-	} else if (err) {
+	if (err) {
 		LOG_ERR("Failed to delete Azure IoT Hub hostname, error: %d", err);
 
```

Comment:
unrelated but nice to do: Could remove these spaces, the rest of the formatting in this file returns right after the log message.

Reply:
Done